pb-jelly/Cargo.toml

```diff
@@ -17,3 +17,13 @@ categories = ["encoding", "parsing", "web-programming"]
 byteorder = "1.3.4"
 bytes = "0.5.6"
 serde = { version = "1.0.114", features = ["derive"] }
+
+[features]
+# Skips UTF-8 validation when using zero-copy Strings
+#
+# The (protobuf specification)[https://developers.google.com/protocol-buffers/docs/proto3#scalar]
+# states that Strings must contain UTF-8 encoded text, so theoretically, validating a chunk
+# of bytes is UTF-8 on deserialization shouldn't be needed. As a precaution though, we do this
+# validation step. But validation can take a relatively large amount of time, so for users
+# who wish to skip this validation, we provide the `zero_copy_string_no_utf8_check` feature flag.
+zero_copy_string_no_utf8_check = []
```

Comment:
Is it possible for cargo bench to test with and w/o this feature flag?
Seems like it might not be possible w/ just how cargo bench works.